packages/docs/src/components/head.js

```diff
@@ -19,14 +19,30 @@ export default props => {
     <Helmet htmlAttributes={{ lang: 'en-US' }}>
       <title>{title}</title>
       <meta name="description" content={pkg.description} />
-      <link rel="icon" type="image/png" href="/icon.png" />
+      <link
+        rel="icon"
+        media="(prefers-color-scheme:dark)"
+        href="/icon-dark.png"
+        type="image/png"
+      />
+      <link
+        rel="icon"
+        media="(prefers-color-scheme:light)"
```

Comment:
Many operating systems don’t support this media query yet, so maybe cut this line so it’s the default?

Reply:
@lachlanjc That is the whole point of this PR, to support `prefers-color-scheme` - I wonder if there is another way of going about doing this 🤔

Reply:
I think @lachlanjc is suggesting moving the default (e.g. light mode) `<link>` above the other one and only using the `media` attribute on the dark mode one – I would assume that would work, but have not tried it out myself

Reply:
Yep, exactly what @jxnblk said 👍